repository/u.json

```diff
@@ -598,10 +598,11 @@
 			{
 			"name": "Urtext",
 			"details": "https://github.com/nbeversl/urtext_sublime",
-			"labels": ["plaintext","notes","writing"],
+			"labels": ["plaintext", "notes", "note-taking", "text", "writing", "zettlekasten"],
 			"releases": [
-				{
-					"sublime_text": ">=3092",
+				{	
+					"platforms": "*",
+					"sublime_text": "*",
```

Comment:
`"sublime_text": ">=3092"` includes ST4 so I wonder why you want a `*` here which basically adds `<3092`.  Also, if you want *all* platforms then just omit the key `platforms`  (as before).   Did you notice something not working?  Maybe *I*'m confused.

Reply:
It appears this PR is unnecessary. All I wanted to do was tag it for compatibility with ST4 as we are trying to get some new test users, _viz_:
<img width="69" alt="Screenshot 2024-07-06 at 7 07 24 AM" src="https://github.com/wbond/package_control_channel/assets/12901567/34a4ca3a-c54f-4c3e-a5a9-1ccd5f16100f">
I just could not find the documentation for that. Thanks for the feedback, and please advise if there is a way to accomplish that.

Reply:
Yeah, the icon is misleading.  It says ST3 but really means ST3+.  🤷  You get the ST4 icon only if you drop ST3 support, e.g. `>=4000` but that's not what you want.  You can still change the labels if you want though.